crypto/whrlpool/wp_block.c

```diff
@@ -75,7 +75,6 @@ typedef unsigned long long u64;
 #   define OPENSSL_SMALL_FOOTPRINT
 #  endif
 #  define GO_FOR_MMX(ctx,inp,num)     do {                    \
-        extern unsigned long OPENSSL_ia32cap_P[];               \
```

Comment:
Does this file need cryptlib.h included also?

Reply:
I did my best to test this, but I'll double-check.

Reply:
looks like the travis picked this up

Reply:
ah, good, then. I was concerned about what happens when ONLY_I386 is defined.
and the variable would exist but initialization not happening.

Reply:
#9582